scripts/check_deps.sh

```diff
@@ -6,7 +6,7 @@ set -e
 pydeps --show-dot --no-show --no-config caikit | grep '\->' > deps.txt
 trap "rm deps.txt" EXIT
 
-if < deps.txt grep -q ".*caikit_runtime.*\->.*caikit_core.*"
+if grep -q ".*caikit_runtime.*\->.*caikit_core.*" deps.txt | grep -v  factories
```

Comment:
I think we _don't_ want this change now, right? I'm guessing this was only to support the circular dependency for registering the runtime finder/initializer inside of `core`?

Reply:
Good point! I just removed it and will verify nothing breaks